backend/src/main/java/gov/cdc/usds/simplereport/config/AuthorizationConfiguration.java

```diff
@@ -419,4 +422,13 @@ public class AuthorizationConfiguration {
           + " && "
           + SPEL_CAN_VIEW_QUEUE_ITEM_FOR_PATIENT)
   public @interface RequirePermissionSubmitTestForPatient {}
+
+  /**
+   * Require the current user to have the {@link UserPermission#SR_CSV_UPLOADER_PILOT} permission
+   * for the CSV Test Result Upload Pilot
+   */
+  @Retention(RUNTIME)
+  @Target(METHOD)
+  @PreAuthorize(SPEL_IS_VALID + " && " + SPEL_HAS_PERMISSION_UPLOADER_PILOT)
+  public @interface RequirePermissionCSVUpload {}
```

Comment:
TIL `@Retention`

...and `@Target`

...and `@interface`. Cool!

Reply:
Are we going to want to grant the new `SR_CSV_UPLOADER_PILOT` permission to [standard users](https://github.com/CDCgov/prime-simplereport/blob/dan/3644-bulk-upload/backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/OrganizationRole.java#L42-L56) and/or [test-entry users](https://github.com/CDCgov/prime-simplereport/blob/dan/3644-bulk-upload/backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/OrganizationRole.java#L35-L41) in addition to org admins? Each of those groups can submit tests in the ordinary fashion but I'm not sure if the intent is to hand over the keys for the bulk upload as well (cc: @zdeveloper).

Reply:
@zdeveloper should we have another `OrganizationRole` for this permission?  Doesn't seem like it belongs in the "Standard User"

Reply:
Mm I think long term we gonna piggy back off the the existing permissions, we're just adding ths permission as a feature flag for the pilot, so I think it's ok to be on the user level, I am ok either way, as long as we can restrict the roll out